src/main/java/org/isf/patient/service/PatientIoOperations.java

```diff
@@ -128,6 +136,7 @@ public Patient getPatient(Integer code) throws OHServiceException {
 			if (isLoadProfilePhotoFromDb) {
```

Comment:
Why is this loaded at the beginning of each file:
<pre>
boolean isLoadProfilePhotoFromDb = LOAD_FROM_DB.equals(GeneralData.PATIENTPHOTOSTORAGE);
</pre>
Can't it be a class static?   It doesn't change unless the system is restarted, correct?